phpstan.neon

```diff
@@ -31,28 +29,7 @@ parameters:
     - "#^Access to an undefined property Drupal\\\\#"
     # PHPUnit deprecation warnings in Drupal 9 that we don't care about.
     - "#^Call to deprecated method setMethods\\(\\) of class PHPUnit\\\\Framework\\\\MockObject\\\\MockBuilder:#"
-    # Other deprecations in Drupal 9 that we can't change because we want to
-    # support Drupal 8.
-    - "#deprecated class Symfony\\\\Component\\\\EventDispatcher\\\\Event#"
-    - "#deprecated class Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseEvent#"
     - "#^Method Symfony\\\\Contracts\\\\EventDispatcher\\\\EventDispatcherInterface\\:\\:dispatch\\(\\) invoked with 2 parameters, 1 required\\.$#"
```

Comment:
can we also remove this line?

Reply:
No, I had that in the first place but had to add it back.